helper/labelSchema.js

```diff
@@ -13,12 +13,17 @@ module.exports = {
   'USA': {
     'borough': getFirstProperty(['borough']),
     'local': getFirstProperty(['locality', 'localadmin', 'county']),
-    'regional': getUsOrCaState,
+    'regional': getUsOrCaOrAusState,
```

Comment:
after how many countries do we rename this to something like `getRegionalValue`? 3 feels like a good time :)